boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_shared.dts

```diff
@@ -0,0 +1,89 @@
+/ {
```

Comment:
Since this file is to be included from other *.dts files, it would be more appropriate to use the .dtsi extension for it. Similarly for nrf5340_audio_dk_nrf5340_shared_sram_planning_conf.dts, nrf5340_audio_dk_nrf5340_cpuapp_common.dts, and nrf5340_audio_dk_nrf5340_cpuapp_partition_conf.dts.